packages/web-components/src/components/va-language-toggle/va-language-toggle.tsx

```diff
@@ -96,30 +96,27 @@ export class VaLanguageToggle {
   }
 
   componentWillLoad() {
-    // always include English
-    const urls = [{
+    this.urls = [{
       label: "English",
       lang: "en",
       href: this.enHref
     }];
 
     if (this.esHref) {
-      urls.push({
+      this.urls = [...this.urls, {
         label: "Español",
         lang: "es",
         href: this.esHref
-      });
+      }];
```

Comment:
This is just a minor refactor related to mutability. By using the spread operator, we can update the `urls` prop without mutating the array.